projects/igniteui-angular/src/lib/splitter/splitter-bar/splitter-bar.component.ts

```diff
@@ -122,4 +137,24 @@ export class IgxSplitBarComponent {
         return !!relatedTabs.find(x => x.resizable === false);
     }
 
+    public onCollapsing(event: any) {
+        const arrowElement = event.srcElement;
```

Comment:
if you send false for left/top and true for right/bottom this can be shortened to something like:
```typescript
public onCollapsing(direction: boolean) {
   const sibling = this.siblings[0];
   this.collapse.emit(sibling.hidden && !direction || !this.pane.hidden && direction ? sibling : this.pane);
}
```

Try it out and see if it works